android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -1722,6 +1719,34 @@ public void onLongPress(MotionEvent e)
 		touchable.setOnTouchListener(new OnTouchListener() {
 			int pointersDown = 0;
 
+			private boolean doRotationEvent(MotionEvent event)
+			{
+				// Calculate the angle between the two fingers
+				float deltaX = event.getX(0) - event.getX(1);
+				float deltaY = event.getY(0) - event.getY(1);
+				double radians = Math.atan(deltaY / deltaX);
+				// Convert to degrees
+				int degrees = (int) (radians * 180 / Math.PI);
```

Comment:
Could use [Math.toDegrees()](https://docs.oracle.com/javase/7/docs/api/java/lang/Math.html#toDegrees(double))

Reply:
:+1: will do these changes tomorrow. Thanks for the feedback